imagery/i.gensig/means.c

```diff
@@ -32,7 +32,7 @@ int compute_means(struct files *files, struct Signature *S)
 	    Rast_get_d_row(files->band_fd[b], cell = files->band_cell[b], row);
 	    for (col = 0; col < ncols; col++) {
 		if (Rast_is_d_null_value(&cell[col])) {
-		    n_nulls++;
+		    n_nulls++; // 'n_nulls' is uninitialized when used here
```

Comment:
I'm afraid that C-style comments are desired - not all CC compilers are able to digest C++-Style comments.
See
https://trac.osgeo.org/grass/wiki/Submitting/C#Comments
- " Don't use the C++ comment style! This confuses several compilers. ..."

If this is outdated as of 2022, I'm fine with that (just the Wiki page should be updated accordingly).

Reply:
Outdated, see:

https://en.cppreference.com/w/c/comment
https://trac.osgeo.org/grass/wiki/RFC/7_LanguageStandardsSupport#CLanguage

Reply:
The comments are there now, only to indicate the warnings.
A proper fix is to be implemented...